src/app/pages/dashboard/widgets/system/common/product-image/product-image.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="product-image-placeholder">
+  <img
+    *ngIf="productImageSrc(); else generic"
+    matTooltipPosition="above"
+    [matTooltip]="'View Enclosure' | translate"
+    [id]="product()"
+    [class.clickable]="hasEnclosureSupport()"
+    [matTooltipDisabled]="!hasEnclosureSupport()"
+    [src]="productImageSrc()"
+    (click)="goToEnclosure()"
+  />
+</div>
+
+<div class="product-image-text">
+  <strong *ngIf="productImage() && isHaLicensed()">(<ng-content></ng-content>)</strong>
+  <strong *ngIf="isUnsupportedHardware()">({{ 'Unsupported Hardware' | translate }})</strong>
+</div>
+
+<ng-template #generic>
+  <ix-icon
+    *ngIf="!isCertified() && !isHaLicensed()"
```

Comment:
Feels like we should have fallback for all the other cases. Currently only two cases are covered.
`!isCertified() && !isHaLicensed()` and `isCertified() && !isHaLicensed()`. If other cases like `isCertified() && isHaLicensed()` or `!isCertified() && isHaLicensed()` are not possible, then never mind. Still, we should use `else` instead of two ifs just in case.